readme.md

```diff
@@ -8,3 +8,22 @@
 
 ## What is Hacktoberfest?
 Hacktoberfest is a month-long celebration of open source software run by DigitalOcean in partnership with GitHub and Twilio. Hacktoberfest is open to everyone in our global community! Four quality pull requests must be submitted to public GitHub repositories. You can sign up anytime between October 1 and October 31.
+
+## Contributors ✨
+
+Thanks goes to these wonderful people 💜
+<table>
+  <tr>
+    <td align="center"><a href="https://github.com/ShehrozIrfan"><img src="https://avatars.githubusercontent.com/u/36946330?s=60&v=4" width="100px;" alt=""/><br /><sub><b>ShehrozIrfan</b></sub></a><br /><a href="#maintenance-Tlazypanda" title="Maintenance">🚧</a></td>
```

Comment:
It's the great work.
But every time there is a new contributor we've to add it manually.
Is there any way it is added automatically?